modules/caddyhttp/reverseproxy/fastcgi/fastcgi.go

```diff
@@ -387,7 +421,7 @@ var headerNameReplacer = strings.NewReplacer(" ", "_", "-", "_")
 
 // Interface guards
 var (
-	_ zapcore.ObjectMarshaler = (*envVars)(nil)
+	_ zapcore.ObjectMarshaler = loggableEnv{}
```

Comment:
I think it should be done like this:

```suggestion
	_ zapcore.ObjectMarshaler = (*loggableEnv)(nil)
```